packages/snaps-utils/src/test-utils/manifest.ts

```diff
@@ -95,6 +101,7 @@ export const getSnapManifest = ({
   description = MOCK_SNAP_DESCRIPTION,
   proposedName = MOCK_SNAP_NAME,
   initialPermissions = MOCK_INITIAL_PERMISSIONS,
+  dynamicPermissions = MOCK_DYNAMIC_PERMISSIONS,
```

Comment:
Do we need/want to use dynamicPermissions in all tests? I think a default value of `undefined` is ok here

Reply:
I removed it for now.